File: 250317/두 사람/two-person.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int a_age, b_age = 0;
    char a_gender, b_gender = ' ';

    cin >> a_age >> a_gender >> b_age >> b_gender;

        if((a_age >= 19 && a_gender == 'M') || (b_age >= 19 && b_gender == 'M')) {
            cout << 1;
        }
        else {
            cout << 0;
        }
    return 0;
}

File: 250317/두 사람/two-person.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a_age = sc.nextInt();
        String a_gender = sc.next();
        int b_age = sc.nextInt();
        String b_gender = sc.next();

        if((a_age >= 19 || b_age >= 19) && (a_gender.charAt(0) == 'M' || b_gender.charAt(0) == 'M')) {
            System.out.println(1);
        }
        else {
            System.out.println(0);
        }
        sc.close();
    }
}